lcr/__init__.py

```diff
@@ -21,17 +23,25 @@
 
 if _LOGGER.getEffectiveLevel() <= logging.DEBUG:
     import http.client as http_client
+
     http_client.HTTPConnection.debuglevel = 1
 
 
 class InvalidCredentialsError(Exception):
     pass
 
 
-class API():
+class API:
     def __init__(
-            self, username, password, unit_number, beta=False,
-            driver=webdriver.Chrome(options=CHROME_OPTIONS)):
+        self,
+        username,
+        password,
+        unit_number,
+        beta=False,
+        driver=webdriver.Chrome(
```

Comment:
This change allow us to use the new version of Selenium and not have to install dev tools manually.